guides/source/active_job_basics.md

```diff
@@ -388,6 +389,23 @@ class GuestsCleanupJob < ApplicationJob
 end
 ```
 
+A callback that `throws :abort` will [halt the callback chain](https://api.rubyonrails.org/classes/ActiveSupport/Callbacks.html)
+and skip the execution of any subsquent before, around and after callbacks:
```

Comment:
How will this affect instrumentation? 

I suspect with a `throw` in a `before_perform`, `enqueue` events will still send but no `discard` or other signal that the job never ran - would that be better handled by raising a custom exception instead, then using `discard_on` it?